website/docs/constructs/cookbook/contributing.md

```diff
@@ -81,7 +81,7 @@ Now you're ready to start contributing to the framework sub project!
     // GIVEN
     const app = new cdk.App();
     const stack = new Stack(app, 'MyStack');
-    const testStack = new TestStack('MskServerkessTestStack', app, stack);
+    const testStack = new TestStack('MskServerlessTestStack', app, stack);
```

Comment:
Can you change this to a generic name instead of `MskServerlessTestStack`

Reply:
Done! I changed by `E2ETestStack`. Is it OK?